packages/commonwealth/client/scripts/views/modals/WelcomeOnboardModal/WelcomeOnboardModal.tsx

```diff
@@ -0,0 +1,56 @@
+import clsx from 'clsx';
+import React, { useState } from 'react';
+import { CWIcon } from '../../components/component_kit/cw_icons/cw_icon';
+import { CWText } from '../../components/component_kit/cw_text';
+import { CWModal } from '../../components/component_kit/new_designs/CWModal';
+import './WelcomeOnboardModal.scss';
+import { PersonalInformationStep } from './steps/PersonalInformationStep';
+
+type WelcomeOnboardModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const WelcomeOnboardModal = ({ isOpen, onClose }: WelcomeOnboardModalProps) => {
+  const [activeStep, setActiveStep] = useState(1);
```

Comment:
can you use some meaningful names instead of numbers for steps?

Reply:
yes, it will be updated when 2nd step is added